packages/core/index.ts

```diff
@@ -100,11 +100,7 @@ export function createEnv<
 ): z.infer<ZodObject<TServer>> & z.infer<ZodObject<TClient>> {
   const runtimeEnv = opts.runtimeEnvStrict ?? opts.runtimeEnv ?? process.env;
 
-  const skip =
-    opts.skipValidation ??
-    (!!process.env.SKIP_ENV_VALIDATION &&
```

Comment:
I believe this is originally from `create-t3-app`, and I don't think we should use this (`process.env.SKIP_ENV_VALIDATION`) anymore, and let the `create-t3-app` mention about this in docs.